apps/generate_non_factory_sboms.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import logging
+import os
+import subprocess
+
+from expandvars import expandvars
+
+from apps.compose_apps import ComposeApps
+from helpers import status
+
+
+def main(args):
+    status("Loading docker-compose files to find non-factory containers")
+    factory_prefix = f"hub.foundries.io/{args.factory}/"
+
+    # We don't need to validate the compose content here - just need to
+    # iterate service images.
+    ComposeApps.App.DockerComposeTool = None
```

Comment:
Is validation causes any issues or there was another reason why you turned it off?

Reply:
it gets noisy when you see the CI output. it starts dumping the compose content to stdout and doesn't make much sense why

Reply:
Probably we need to add `--quite` flag` to the `config` cmd.

Reply:
https://github.com/foundriesio/ci-scripts/pull/208/commits/19c68c605664807fc8dca2905a2752d8eb8d3657

Reply:
i've done another force push to use your suggestion and have a new ci container. everything should be in place now